### PR TITLE
Set up and prepare my portfolio

### PR DESCRIPTION
**Pull request by developerbaalla**
set up a new repository and prepare it
build the first 2 sections of the mobile website

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Hello Microverse
```
Comment (gtekle) [inline README.md]: - [x] You should update the title of the project to the new project. e.g. Portfolio, My Portfolio, etc

File: README.md [hunk 2]
```diff
@@ -1,69 +1,40 @@
...
 
-> Hello Microverse! project.
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices i have learned about in previous steps. I will be using them in all Microverse projects and most likely in my future job as well.
```
Comment (gtekle) [inline README.md]: - [x] You should update the description section to reflect the new project. I believe this description is for the `Hello Microverse` project.

File: README.md [hunk 3]
```diff
@@ -1,69 +1,40 @@
...
+## Setup
+git clone https://github.com/developerbaalla/Hello-Microverse.git
 
+cd Hello-Microverse
 
+code.
 
 ## Authors
 
 👤 **Author1**
```
Comment (gtekle) [inline README.md]: - [x] You should use the author name in place of the placeholder text **Author1**.

File: css/port.css [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  background-color: #fff;
+}
+
+.bk_container {
+  background-color: #e5e5e5;
+  max-width: 375px;
```
Comment (gtekle) [inline css/port.css]: - [x] You should remove the `max-width` because it is making your website not responsive.

File: css/port.css [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+  margin: auto;
+}
+
+.port_container {
+  max-width: 375px;
```
Comment (gtekle) [inline css/port.css]: - [x] You should remove the `max-width` because it is making your website not responsive.

File: css/port.css [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+  height: 100vh;
+  background-image: url('../images/header-shapes\ mobile.png');
+  background-color: #fff;
```
Comment (gtekle) [inline css/port.css]: - [ Optional ] You may add `background-size: cover;` to make the background image cover the full width.

File: css/port.css [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+
+.bk_container {
+  background-color: #e5e5e5;
+  max-width: 375px;
+  margin: auto;
+}
+
+.port_container {
+  max-width: 375px;
+  height: 100vh;
```
Comment (gtekle) [inline css/port.css:21]: - [ ] If at all you have to use fixed height, you should use `min-height: 100vh` instead. This way, it make the section to be at least 100vh if there is no content, but as more content are added it allows the section to grow downwards.

File: css/port.css [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+  border-bottom-left-radius: 100px;
+  padding-top: 10px;
+}
+
+.header {
+  padding: 0 35px;
+}
+
+.container {
+  width: 327px;
```
Comment (gtekle) [inline css/port.css]: - [x] Please remove this fixed width for it helps to get responsive website.

File: index.html [hunk 9]
```diff
@@ -1,24 +1,60 @@
...
+  <title>Mohamed Portfolio</title>
 </head>
 <body>
-    <header>
-        <div>
-            <img src="src/img/logo.png" class="logo" alt="Portfolio">
-            <img src="src/img/menu-button.png" class="menu-button" alt="Menu">
-        </div>
+  <!--header-->
+ <div class="bk_container">
```
Comment (gtekle) [inline index.html]: - [ ] You should apply the background image on the `headline` section and it doesn't include the `toolbar/header` section.

- [ ] You should use semantically meaningful sections. For example, 
```
<body>
   <header>
       logo
       Navbar
   </header>
   <section class='headline'>
    <div class='container' >
          ......
     </div>
   </section>
</body>
```
If you have to use a container for the purpose of apply padding, margins, and background images, you can make it within the semantically meaningful section elements such `<header>, <section>` as in above outline.

File: index.html [hunk 10]
```diff
@@ -1,24 +1,60 @@
...
+  <div class="port_container">
+    <header class="header">
+      <nav class="navbar">
+        <a href="#" class="logo">
+          <img src="./images/Portfolio_Logo.png" alt="My Logo"/></a>
```
Comment (gtekle) [inline index.html]: - [x] You should use your name or something else for the logo instead of just `My Logo`.

File: index.html [hunk 11]
```diff
@@ -1,24 +1,60 @@
...
-    <header>
-        <div>
-            <img src="src/img/logo.png" class="logo" alt="Portfolio">
-            <img src="src/img/menu-button.png" class="menu-button" alt="Menu">
-        </div>
+  <!--header-->
+ <div class="bk_container">
+  <div class="port_container">
+    <header class="header">
+      <nav class="navbar">
```
Comment (gtekle) [inline index.html]: - [ ] You should `Flexbox` to layout contents of the both the `header` and the `headline` sections as per the project requirement. `In order to lay out the elements on the page you should use Flexbox in all 2 sections`

File: index.html [hunk 12]
```diff
@@ -1,24 +1,60 @@
...
     </header>
-    <section class="headline">
-        <h2>I’m Mohamed Glad to see you!</h2>
-        <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-        <p class="letsconnect">LET’S CONNECT</p>
-        <img src="src/img/social-media-icons.png" class="social-media-icons" alt="Social Media">
+   <!--main section--> 
+    <section id="main_section" class="grid" >
+     <div class="container">
+      <h1 class="h1">I’m Mohamed Baalla Glad to see you!</h1>
```
Comment (gtekle) [inline index.html]: - [x] Nice to have used `<h1>` here for this is the most important heading you have in the whole Portfolio website. However, let's split the `Glad to see you!` text as a sub-heading from the main heading and wrapping with `<h2>, <h3>, .. <h6>`

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @developerbaalla :wave:, 

You have done a splendid job so far! :clap: :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there-01


## Highlights
:heavy_check_mark: Nice UI
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Professional ReadMe

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please check the inline comments below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Hello @developerbaalla  😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

[Optional]

- [ ] For future projects when a Code Reviewer asks for some changes Kindly try to make a commit for each issue your resolve instead of putting everything in a single commit. It's really common in tech companies where you'll be working in the future soon! 😃 

## Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**